samples/tab-sso/nodejs/.gitignore

```diff
@@ -63,5 +63,24 @@ dist/
 manifest/
 build/
 
-# VS Code config files
-.vscode/**
\ No newline at end of file
+# # # # # # # # # # # # # # 
```

Comment:
Please refer to https://github.com/formulahendry/teamsfx-migrator/blob/main/templates/tab-sso/.gitignore.example#LL68C1-L84C5

Reply:
added changes